src/main/resources/digital/slovensko/autogram/server/server.yml

```diff
@@ -366,15 +366,27 @@ components:
           example: '<?xml version="1.0"?><xsl:stylesheet version="1.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"><xsl:template match = "/"><h1><xsl:value-of select="/Document/Title"/></h1></xsl:template></xsl:stylesheet>'
           description: "Optional XML transformation used to . Format is dictated by `payloadMimeType`."
 
+        previewWidth:
```

Comment:
dajme tu asi `prefWidth` preview sa mi s nicim nespaja, ono je to vizualizacia - nie nahlad.

Reply:
@celuchmarek cize nechame `prefPreviewWidth` ?